tests/baselines/reference/jsdocPrefixPostfixParsing.errors.txt

```diff
@@ -0,0 +1,46 @@
+tests/cases/conformance/jsdoc/prefixPostfix.js(8,13): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(9,12): error TS1014: A rest parameter must be last in a parameter list.
+tests/cases/conformance/jsdoc/prefixPostfix.js(10,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(11,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(12,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(13,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(14,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+tests/cases/conformance/jsdoc/prefixPostfix.js(15,12): error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+
+
+==== tests/cases/conformance/jsdoc/prefixPostfix.js (8 errors) ====
+    /**
+     * @param {number![]} x - number[]
+     * @param {!number[]} y - number[]
+     * @param {(number[])!} z - number[]
+     * @param {number?[]} a - (number | null)[]
+     * @param {?number[]} b - number[] | null
+     * @param {(number[])?} c - number[] | null
+     * @param {?...number} d - number[] | null
+                ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...?number} e - (number | null)[]
+               ~~~~~~~~~~
+!!! error TS1014: A rest parameter must be last in a parameter list.
+     * @param {...number?} f - number[] | null
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...number!?} g - number[] | null
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...number?!} h - number[] | null
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...number[]} i - number[][]
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...number![]?} j - number[][] | null
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.
+     * @param {...number?[]!} k - (number[] | null)[]
+               ~~~~~~~~~
+!!! error TS8028: JSDoc '...' may only appear in the last parameter of a signature.

```

Comment:
But this is the last parameter of the signature.

Reply:
Also, what is the error for repeated ellipses? It used to mean n-dimensional arrays, so it should definitely be an error now.

Reply:
I think the error is because this is parsed as `(((...number)?)!)[]`. I've added a commit that changes the parser so that `...` should be at the top level.